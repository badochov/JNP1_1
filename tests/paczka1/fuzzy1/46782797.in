t4VTPe S671 99,4
y0Y2 S671 97,3
y0Y2 S671 65,0
?
EykA8DPNe A317 79,3
?A317
t4VTPe A578 97,0
EykA8DPNe S487 17,7
EykA8DPNe S487 80,0
?

EykA8DPNe A317 59,8
t4VTPe A317 38,8

y0Y2 S671 235,3
EykA8DPNe A317 220,1
t4VTPe A317 177,7
?mt6
mt6 A317 99,4
?S671
mt6 A317 232,6
t4VTPe A317 63,6
mt6 A317 92,9
Bo6dgypx S487 15,5
mt6 A317 65,0
t4VTPe A317 21,3
y0Y2 S671 59,4
EykA8DPNe A317 69,8
y0Y2 S671 172,9
RRb3DQu S487 39,2

S487 S487 1,3
S487 S487 3,6
EykA8DPNe A317 197,5
y0Y2 S671 99,5
y0Y2 S671 216,9
Bo6dgypx S487 70,8
EykA8DPNe A578 8,9
?
mt6 S487 76,5