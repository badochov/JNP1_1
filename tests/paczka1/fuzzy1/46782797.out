y0Y2 S 32,3
S671 32,3
EykA8DPNe S 62,3
y0Y2 S 32,3
S487 62,3
S671 32,3
S671 32,3
Bo6dgypx S 55,3
EykA8DPNe A 288,0 S 62,3
S487 S 2,3
mt6 A 161,1
t4VTPe A 181,2
y0Y2 S 281,6
A317 630,3
S487 119,9
S671 281,6
